api/src/main/java/org/openmrs/api/impl/GlobalLocaleList.java

```diff
@@ -33,17 +33,10 @@
 	public void globalPropertyChanged(GlobalProperty newValue) {
 		allowedLocales = new LinkedHashSet<Locale>();
 		for (String allowedLocaleString : newValue.getPropertyValue().split(",")) {
-			try {
+		//The try catch blocked has been removed.
 				Locale allowedLocale = LocaleUtility.fromSpecification(allowedLocaleString.trim());
 				if (allowedLocale != null) {
-					allowedLocales.add(allowedLocale);
-				}
-			}
-			catch (Exception e) {
-				// bad locale spec? just ignore it. the UI should take care of
-				// guiding the user.
-			}
-		}
+					allowedLocales.add(allowedLocale);}
```

Comment:
Did you format this code? The closing bracket should have gone to the next line if you did